demonstrations/tutorial_adversarial_attacks_QML.metadata.json

```
@@ -0,0 +1,53 @@
+{
+    "title": "Adversarial Attacks on Quantum Machine Learning",
+    "authors": [
+        {
+            "id": "maxi_wendlinger"
+        },
+        {
+            "id": "kilian_tscharke"
+        }
+    ],
+    "dateOfPublication": "",
+    "dateOfLastModification": "",

```

```suggestion
    "dateOfPublication": "2024-12-31T00:00:00+00:00",
    "dateOfLastModification": "2024-12-31T00:00:00+00:00",
```
@mwendlinger , let's just add a placeholder date here, so the metadata checker doesn't complain. :)